docs/en/ingest-management/elastic-agent/configuration/env/shared-env.asciidoc

```diff
@@ -110,6 +112,23 @@ When set to `1`, this automatically forces {fleet} enrollment as well.
 
 // =============================================================================
 
+// tag::fleet-setup[]
+
+[id="env-{type}-fleet-setup"]
+`FLEET_SETUP`
```

Comment:
@dedemorton,  we were actually missing `FLEET_SETUP`. I've added it with the deprecation notice